test/Ben.Demystifier.Test/MethodTests.cs

```diff
@@ -123,10 +123,15 @@ private void MethodWithLambda()
             action();
         }
 
-        private void MethodWithAsyncLambda()
+        private async Task MethodWithAsyncLambda()
         {
-            Func<Task> action = async () => throw new ArgumentException();
-            action().ConfigureAwait(false).GetAwaiter().GetResult();
+            Func<Task> action = async () =>
+            {
+                await Task.Delay(0);
```

Comment:
Curious:

Why not `Task.Yield()` ?

Reply:
Probably should have done `await Task.CompletedTask`; its just to stop the CI complaining about no await in the method :)